### ISSUE TITLE
Having issues with Paypal MPL sandbox. Is MPL support being taken off? 

### ISSUE THREAD

**Issue by CGTGauravk** [paypal/sdk-packages#73]
Hello,

We have been facing an issue in launching paypal instance for preapproval, since today morning.
We are facing this issue in both, iOS & Android.

Is the MPL support being removed?

=======================================================
Following is the exception that we get in Android

06-02 19:09:07.991: E/NetworkHandler(8703): postXML caught exception doing I/O, Connection to https://mobileclient.sandbox.paypal.com refused
06-02 19:09:07.992: W/System.err(8703): org.apache.http.conn.HttpHostConnectException: Connection to https://mobileclient.sandbox.paypal.com refused
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:193)
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:169)
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:124)
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:366)
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:560)
06-02 19:09:07.992: W/System.err(8703): 	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:492)
06-02 19:09:07.993: W/System.err(8703): 	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:470)
06-02 19:09:07.993: W/System.err(8703): 	at com.paypal.android.c.f.a(SourceFile:1759)
06-02 19:09:07.993: W/System.err(8703): 	at com.paypal.android.c.f.B(SourceFile:1866)
06-02 19:09:07.993: W/System.err(8703): 	at com.paypal.android.c.f.d(SourceFile:94)
06-02 19:09:07.993: W/System.err(8703): 	at com.paypal.android.c.f$3.run(SourceFile:566)
06-02 19:09:07.993: W/System.err(8703): Caused by: java.net.ConnectException: failed to connect to /173.0.82.101 (port 443): connect failed: ETIMEDOUT (Connection timed out)
06-02 19:09:07.994: W/System.err(8703): 	at libcore.io.IoBridge.connect(IoBridge.java:124)
06-02 19:09:07.994: W/System.err(8703): 	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:183)
06-02 19:09:07.994: W/System.err(8703): 	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:452)
06-02 19:09:07.994: W/System.err(8703): 	at java.net.Socket.connect(Socket.java:884)
06-02 19:09:07.994: W/System.err(8703): 	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:124)
06-02 19:09:07.994: W/System.err(8703): 	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:149)
06-02 19:09:07.994: W/System.err(8703): 	... 10 more
06-02 19:09:07.994: W/System.err(8703): Caused by: android.system.ErrnoException: connect failed: ETIMEDOUT (Connection timed out)
06-02 19:09:07.997: W/System.err(8703): 	at libcore.io.Posix.connect(Native Method)
06-02 19:09:07.997: W/System.err(8703): 	at libcore.io.BlockGuardOs.connect(BlockGuardOs.java:111)
06-02 19:09:07.997: W/System.err(8703): 	at libcore.io.IoBridge.connectErrno(IoBridge.java:137)
06-02 19:09:08.003: W/System.err(8703): 	at libcore.io.IoBridge.connect(IoBridge.java:122)
=======================================================

Hoping to hear soon.
Thanks in advance for any help.

**Comment by bluk** [comment]
Hi, this connection error I believe was resolved but I see you are now encountering #74 . I'll close this issue, but if you encounter this again after #74 is resolved, then please leave a comment or open a new issue. Thanks.
